packages/kit/test/apps/basics/test/test.js

```diff
@@ -1043,15 +1043,27 @@ test.describe('$app/paths', () => {
 		);
 	});
 
-	test('replaces %sveltekit.assets% in template with relative path', async ({ page }) => {
+	test('replaces %sveltekit.assets% in template with relative path', async ({
```

Comment:
I'm guessing you meant absolute instead of relative? should we include a bit of a comment here as well?
```suggestion
  // some browsers will re-request assets after a `pushState` 
  // https://github.com/sveltejs/kit/issues/3748#issuecomment-1125980897
	test('replaces %sveltekit.assets% in template with absolute path', async ({
```

Reply:
no, `%sveltekit.assets%` is relative. that wasn't changed by this PR

Reply:
oh, I guess you're talking about the server-side replacement. I was thinking of the client-side replacement that was just introduced. I wonder if we should just say `properly handles %sveltekit.assets% in template` or something

Reply:
```suggestion
	// some browsers will re-request assets after a `pushState`
	// https://github.com/sveltejs/kit/issues/3748#issuecomment-1125980897
	test('replaces %sveltekit.assets% in template with relative path, and makes it absolute in the client', async ({
```